### ISSUE TITLE
plot,section-method should give unit for sigma-theta panel (and notice EOS, too)

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1341]
It gives a unit for temperature but not sigma-theta.

**Comment by dankelley** [comment]
Also, it doesn't notice the `eos` value, so it can switch from $\sigma_\theta$ to $\sigma_0$.

**Comment by dankelley** [comment]
Fixed in "develop" branch commit 41cae93c8d9374d7ad79987fc2c74bbd362ecefc. Note that the sigma-0 contours for the `eos="gsw"` case are unlikely to look different from the sigma-theta ones for the `eos="unesco"` case. I did a manual check of the calculation for station 10 of `data(section)` manually and found as below (a quote from the code)
```R
## The contour will probably look very much like for the "unesco" case,
## apart from the different legend. I say this because I used station 10
## of data(section) as a test case, and found that the RMS difference
## between results computed with the two formulations to be 0.005kg/m^3,
## or just 0.02% of the mean value.
```
